R/fb_ad.R

```diff
@@ -12,6 +13,7 @@ fbad_create_ad <- function(fbacc,
                            name,
                            adset_id,
                            creative_id,
+                           campaign_id, 
```

Comment:
Only Adset ID is required by Facebook so instead of requiring the campaign ID here `!missing(campaign_id)` can be removed from `stopifnot`